ivadomed/transforms.py

```diff
@@ -152,23 +143,24 @@ def __call__(self, sample):
         input_data = sample['input']
         roi_data = sample['roi']
 
-        w, h = input_data.size
-        tw, th = self.size
-        th_half, tw_half = int(round(th / 2.)), int(round(tw / 2.))
-
         # compute center of mass of the ROI
         x_roi, y_roi = center_of_mass(np.array(roi_data).astype(np.int))
         x_roi, y_roi = int(round(x_roi)), int(round(y_roi))
 
-        # compute top left corner of the crop area
-        fh = y_roi - th_half
-        fw = x_roi - tw_half
+        for i in range(len(input_data)):
+            w, h = input_data[i].size
+            tw, th = self.size
+            th_half, tw_half = int(round(th / 2.)), int(round(tw / 2.))
+
+            # compute top left corner of the crop area
+            fh = y_roi - th_half
+            fw = x_roi - tw_half
```

Comment:
Can go out of the loop (I'm in a fancy mood ;) ).

Reply:
Like [this](https://github.com/neuropoly/ivado-medical-imaging/blob/ab/Missing_Modality/ivadomed/transforms.py#L152-L162)? 😃